src/test/java/de/azapps/kafkabackup/common/record/RecordJSONDeserializerTest.java

```diff
@@ -0,0 +1,51 @@
+package de.azapps.kafkabackup.common.record;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import de.azapps.kafkabackup.common.record.Record;
+
+import static org.junit.Assert.assertEquals;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.InputStream;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.Files;
+import java.util.Base64;
+import java.util.Comparator;
+
+public class RecordJSONDeserializerTest {
+
+    @Test
+    public void deserializeTest() throws Exception {
```

Comment:
there is also some good convention for writing tests - I mean about using "given/when/then" sections
https://www.martinfowler.com/bliki/GivenWhenThen.html

can we use it, what do you think?